data/changelog/odoc/2021-10-06-odoc-2.0.0.md

```diff
@@ -55,7 +55,7 @@ changelog: |
 Hot on the heels of the OCaml 4.13 announcement(s!), the `odoc` team is pleased to announce the release of `odoc 2.0.0`!
 
 *tl;dr:*
-The new version produces [much better output](https://ocaml-doc.github.io/odoc-examples/core_kernel/Core_kernel/Bigbuffer/index.html) than [the old version](https://ocaml.janestreet.com/ocaml-core/latest/doc/core_kernel/Core_kernel/Bigbuffer/index.html), it's the engine at the core of the package docs in [v3.ocaml.org](https://v3.ocaml.org/packages), and it also has a [new website](https://ocaml.github.io/odoc).
+The new version produces [much better output](https://ocaml-doc.github.io/odoc-examples/core_kernel/Core_kernel/Bigbuffer/index.html) than [the old version](https://ocaml.janestreet.com/ocaml-core/latest/doc/core_kernel/Core_kernel/Bigbuffer/index.html - [404 Not Found]), it's the engine at the core of the package docs in [v3.ocaml.org](https://v3.ocaml.org/packages), and it also has a [new website](https://ocaml.github.io/odoc).
```

Comment:
```suggestion
The new version produces [much better output](https://ocaml-doc.github.io/odoc-examples/core_kernel/Core_kernel/index.html) than [the old version](https://v3.ocaml.org/p/core_kernel/v0.15.0), it's the engine at the core of the package docs in [v3.ocaml.org](https://v3.ocaml.org/packages), and it also has a [new website](https://ocaml.github.io/odoc).
```
Although the first link didn't show a 404 error in this bot review, it was broken when I tried to go to it. This suggested link worked.

Second link: Jane Street has a note that their docs moved to the v3 site